samples/holiday-package-sample/README.md

```diff
@@ -0,0 +1,112 @@
+# Online Holiday Packages Booking System.
+
+This scenario is about an online Holiday Package Booking System.
+This is a sample which offers a quick reservation system which involves minimum efforts. This sample will show how
+WSO2 EI is capable of considering a mixture of numerous holiday properties (Flight/ Car/ Hotel) to provide the client with the most suitable holiday package.
```

Comment:
I think this description is not correct, we need to explain the scenario first without mentioning about EI, and then describe how we can implement the scenario using EI.